js/apps/system/_api/gharial/APP/gharial.js

```diff
@@ -762,7 +762,7 @@ router.post('/:graph/edge/:collection', function (req, res) {
 
   // _to vertex
   try {
-    db._document(req.body._to);
+    db._exists(req.body._to);
```

Comment:
`_exists` will return `false` instead of throwing an error (unless `_rev` check fails which is impossible here)
So the logic is now broken